Source/Core/Core/Core.cpp

```diff
@@ -62,6 +62,15 @@
 #include "VideoCommon/OnScreenDisplay.h"
 #include "VideoCommon/VideoBackendBase.h"
 
+//This can be removed when we move to VS2015
```

Comment:
This isn't completely true anymore, right? Android still needs specific code even if VS doesn't.

Also, we generally use a space between the slashes and the comment text.